docker-compose.yml

```diff
@@ -0,0 +1,49 @@
+version: '3'
+
+services:
+  web:
+    build:
+      context: .
+      dockerfile: Dockerfile-dev
+    image: pmg_flask
+    depends_on:
+      - postgres
+      - elastic
+    volumes:
+      - .:/app
+    environment:
+      - SQLALCHEMY_DATABASE_URI=postgresql://pmg:pmg@postgres/pmg?client_encoding=utf8
+      - ES_SERVER=http://elastic:9200
+    ports:
+      - "5000:5000"
+    command: python app.py runserver
+    extra_hosts:
+      - "api.pmg.test:127.0.0.1"
+      - "pmg.test:127.0.0.1"
```

Comment:
I think this is actually not needed. It just needs api.pmg.test to be able to speak to itself
```suggestion
```